### ISSUE TITLE
URL of iOS App: connection ro server failed

### ISSUE THREAD

**Issue by pedroct** [supergeorg/Grocy-SwiftUI#14]
I installed the app on my iPhone 11 with no problems.
I configured the `BASE_URL` in the **config.php** file with my domain http://estoque.pedroct.com.br , but when trying to access, it shows the following error message/image:
When I use the manuall connection in the web browser, it works.

![grocy_erro](https://user-images.githubusercontent.com/5846288/121086077-17605b00-c7b9-11eb-927c-d06b4a3d0996.jpg)

**Comment by supergeorg** [comment]
This seems to be connected to the usage of a non-https server.
I have added some more logging to the app to identify this possible error.

**Comment by yvansandoz** [comment]
Similar problem on my side. I run Grocy on my Home Assistant server all behind reversed proxy server, each with its own port (HA: 8123, Grocy: 9192). Everything works fine but the connection from the iOS app with the QR code connection. Where should I look to fix this problem?

**Comment by supergeorg** [comment]
If direct login entry works, maybe the `BASE_URL` is misconfigured. Test the QR code with another app (e.g. iOS Camera) and identify if the login URL is correct (not missing the server address and port).

**Comment by yvansandoz** [comment]
Yes, the BASE_URL is wrong, but going through the different descriptions, I cannot figure out how to set it up.
